SDWebImage/MapKit/MKAnnotationView+WebCache.h

```diff
@@ -53,6 +53,22 @@
           placeholderImage:(nullable UIImage *)placeholder
                    options:(SDWebImageOptions)options NS_REFINED_FOR_SWIFT;
 
+/**
+ * Set the imageView `image` with an `url`, placeholder, custom options and context.
+ *
+ * The download is asynchronous and cached.
+ *
+ * @param url         The url for the image.
+ * @param placeholder The image to be set initially, until the image request finishes.
+ * @param options     The options to use when downloading the image. @see SDWebImageOptions for the possible values.
+ * @param context     A context contains different options to perform specify changes or processes, see `SDWebImageContextOption`. This hold the extra objects which `options` enum can not hold.
+ */
+
+- (void)sd_setImageWithURL:(nullable NSURL *)url
+          placeholderImage:(nullable UIImage *)placeholder
+                   options:(SDWebImageOptions)options
+                   context:(nullable SDWebImageContext *)context;
```

Comment:
This one, since `SDWebImageContext` was bridge to Swift as` [SDWebImageContextOption : Any]?`, which does not contains a default arg. So we shouldn't use `NS_REFINED_FOR_SWIFT` here.